integration_tests/steps/integration_tests.py

```diff
@@ -278,6 +278,16 @@ def run_sim_console(context, scenario, hours):
               .format(export_path=context.export_path, scenario=scenario, hours=hours))
 
 
+@when('we run the d3a simulation on console with {scenario} for {hours} hrs '
+      'with default tick/slot length')
+def run_sim_console_decreased_tick_slot_length(context, scenario, hours):
+    context.export_path = os.path.join(context.simdir, scenario)
+    os.makedirs(context.export_path, exist_ok=True)
+    os.system("d3a -l FATAL run -d {hours}h -t 40s -s 40m --setup={scenario} "
+              "--export-path={export_path}"
+              .format(export_path=context.export_path, scenario=scenario, hours=hours))
+
+
```

Comment:
As this is a copy of the code above these lines apart from the slot and tick length, I suggest to parse the slot and tick length in the step caption. 
Is the unusual slot and tick length needed here to test a corner case?

Reply:
Good idea, I'll do so, thanks. It is actually required in order for the 2-sided market offers-bids to match for the setup files that I used (essentially used the same setup files with the one sided market and adapted some parameters for 2 sided market to work exactly like the one sided). 40 is the max slot and tick length I could use actually. Any value lower than that works, but delays unnecessarily the simulation.